edx_proctoring/urls.py

```diff
@@ -9,6 +9,8 @@
 
 app_name = u'edx_proctoring'
 
+CONTENT_ID_PATERN = r'(?P<content_id>([A-z0-9]+|(?:i4x://?[^/]+/[^/]+/[^/]+/[^@]+(?:@[^/]+)?)|(?:[^/]+)))'
```

Comment:
fix a typo, please
```suggestion
CONTENT_ID_PATTERN = r'(?P<content_id>([A-z0-9]+|(?:i4x://?[^/]+/[^/]+/[^/]+/[^@]+(?:@[^/]+)?)|(?:[^/]+)))'
```

Reply:
humm it would be nice to use the existing USAGE_ID pattern since we have it. But that's sort of an odd settings dependency... I think the way you have it makes sense for now